.github/workflows/update-as.yml

```
@@ -0,0 +1,92 @@
+      # note that we commit here
+      - name: Update the submodule to the release application services
+        if: env.ALREADY_CREATED == 'false'
+        run: |
+          cd external/application-services
+          echo "$(git remote -v)"
+          git fetch --all --tags
+          git checkout $AS_VERSION
+          cd ../..
+          git commit -am "Updates application services submodule to $AS_VERSION"
+      - name: Update Package.swift with new xcframework URL and checksum
+        if: env.ALREADY_CREATED == 'false'
+        run: |
+          python automation/update_package_swift.py
+          git commit -am "Updates Package.swift with $AS_VERSION release"
+          
+      - name: Install Rust for the make_tag script
+        if: env.ALREADY_CREATED == 'false'
+        run: |
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+          source $HOME/.cargo/env
+      - name: Install swiftformat to format the uniffi generated bindings
+        if: env.ALREADY_CREATED == 'false'
+        uses: Cyberbeni/install-swift-tool@v2
+        with:
+          url: https://github.com/nicklockwood/SwiftFormat
+      # We strip away the `v` in the AS Version and run the make_tag
+      # script, which will generate all the code that needs to be generated
+      # Note that the tag argument isn't useful here, since we will
+      # have to cut a release with a new tag (possibly with the same name)
+      # after the PR is merged
+      - name: Runs make_tag script
+        if: env.ALREADY_CREATED == 'false'
+        run: |
+          ./make_tag.sh ${AS_VERSION:1}
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        if: env.ALREADY_CREATED == 'false'
+        with:
+          commit-message: Auto update with latest AS Release ${{ env.AS_VERSION }}
+          title: Auto Update with latest AS Release ${{ env.AS_VERSION }}
+          branch: update-as-to-${{ env.AS_VERSION }}
```

there should be a comment here making sure that if someone changes the branch name here, they change it in the `is_already_updated`, shouldn't be a big problem if they don't, but better if we fail early if the branch already existed. Otherwise the jobs will go on until it's about to create the PR